.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,55 @@
+<!--- ↑ Provide a general summary of your changes in the Title above ↑ -->
+
+## Description
+
+<!--- Describe your changes in detail -->
+
+## Related Issue
+
+<!--- This project only accepts pull requests related to open issues -->
+<!--- If suggesting a new feature or change, please discuss it in an -->
+<!--- issue first -->
+<!--- If fixing a bug, there should be an issue describing it with -->
+<!--- steps to reproduce -->
+<!--- Please link to the issue here with the prefix or resolve -->
+<!--- keyword -->
+
+## Motivation and Context
+
+<!--- Why is this change required? What problem does it solve? -->
+
+## How Has This Been Tested?
+
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests -->
+<!--- you ran to see how your change affects other areas of the -->
+<!--- code, etc. -->
+
+## Screenshots
+
+<!--- if appropriate, otherwise the section can be removed -->
+
+## Types of changes
+
+<!--- What types of changes does your code introduce? Stroke -->
+<!-- (i.e. ~stroked text~) the ones that don't apply: -->
+- Chore (non-breaking change which refactors / improves the existing code base)
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Breaking change (fix or feature that would cause existing functionality to
+  change)
+
+## Checklist:
+
+<!--- Go over all the following points, and replace the `:red_circle:` in all -->
+<!--- the lines with a :white_check_mark: when relevant. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're -->
+<!--- here to help! -->
+- :red_circle: My code follows the code style of this project.
+- :red_circle: My change requires a change to the documentation.
+- :red_circle: I have updated the documentation accordingly.
+- :red_circle: I have read the [**CONTRIBUTING**][CONTRIBUTING_FILE] document.
+- :red_circle: I have added tests to cover my changes.
+- :red_circle: All new and existing tests passed.
```

Comment:
You should use this checklist https://github.com/fewlinesco/node-web-libraries/blob/main/.github/PULL_REQUEST_TEMPLATE.md/

Reply:
I prefer it to. Faster to look at or change something after.

Reply:
![image](https://user-images.githubusercontent.com/29123047/117166629-d72b4880-adc6-11eb-9c95-755919bf4e65.png)
😏

Reply:
Hum, my bad. I took the Sparta version (that we should maybe change then ?)

Reply:
`sparta-monorepo` and `connect-js` are not the same kind, so I'd say no. I mean, you'd still have to provide screenshots, but we don't do versionning on `sparta-monorepo`.

Reply:
Oh, I was talking about the circles vs the checks 👍